odl/tomo/analytic/filtered_back_projection.py

```diff
@@ -163,19 +169,18 @@ def tam_danielson_window(ray_trafo, smoothing_width=0.05, n_half_rot=1):
     dx = (rot_dir[0] * ray_trafo.range.meshgrid[1] +
           rot_dir[1] * ray_trafo.range.meshgrid[2])
 
-    # Compute angles
-    phi = np.arctan(dx / (src_radius + det_radius))
-    theta = phi * 2
+    dx_axis = dx * src_radius / (src_radius + det_radius)
 
-    # Compute lower and upper bound
-    source_to_line_distance = src_radius + src_radius * np.cos(theta)
-    scale = (src_radius + det_radius) / source_to_line_distance
+    def Vn(u):
+        return (pitch / (2 * np.pi) *
+                (1 + (u / src_radius) ** 2) *
+                (n_pi * np.pi / 2.0 - np.arctan(u / src_radius)))
```

Comment:
Not sure I wanna go read the paper to see what this function does. Is there some intuition behind it? A comment or a docstring would help here.

Reply:
Frankly, I dunno. The old code was me with pen and paper (I didn't even know the method existed and had a name at that time), trying to draw in 3D.

Interestingly, it seems that this change does not do anything to the output of the function, but merely harmonizes it with the article.

Reply:
I can add the general idea that this is the helix that the source travels on, projected onto the detector.

Reply:
> I can add the general idea that this is the helix that the source travels on, projected onto the detector.

That'd be a great help!